docs/articles/documentation/test-api/selecting-page-elements/selectors/selector-options.md

```diff
@@ -11,12 +11,13 @@ redirect_from:
 You can pass the following options to the [Selector constructor](creating-selectors.md).
 
 * [options.boundTestRun](#optionsboundtestrun)
+* [options.dependencies](#optionsdependencies)
```

Comment:
(Just curiosity) AFAIR we removed `dependencies`  on purpose, now we restore them by so reasons? @AndreyBelym

Reply:
Looks like we wanted to force customers using the Functional Style Selectors instead of passing other selectors as deps:

https://github.com/DevExpress/testcafe/commit/86aa0999c12ff3bf87af64b678a90312cfb7400e#diff-cf8140e0e7c9167a1784410da2f6bb60R52

But we wanted to document it for another use case: when a selector's library author wants to incorporate some state into selectors:

https://github.com/DevExpress/testcafe/issues/3338

Reply:
So I think it's worth re-documenting